connect-examples/oauth/python-aws-chalice/README.md

```diff
@@ -10,81 +11,77 @@ comments included in this project.
-    [Square Developer Dashboard](https://developer.squareup.com/apps).
+- Add a new environment variable `application_id`, and fill value with your sandbox application's ID, which is available on your
+  [Square Developer Dashboard](https://developer.squareup.com/apps).
 
-  * Add a new environment variable `application_secret`, and fill value with your sandbox application's secret, also available on your
-    [Square Developer Dashboard](https://developer.squareup.com/apps).
+- Add a new environment variable `application_secret`, and fill value with your sandbox application's secret, also available on your
+  [Square Developer Dashboard](https://developer.squareup.com/apps).
 
-  * Add a new environment variable `token_encrypt_secret` with the key value which is generated by following python script:
+- Add a new environment variable `token_encrypt_secret` with the key value which is generated by following python script:
 
-      ```Python
-      >>> from cryptography.fernet import Fernet
-      >>> key = Fernet.generate_key()
-      >>> print(key.decode())
-      ```
+  ```Python
+  >>> from cryptography.fernet import Fernet
+  >>> key = Fernet.generate_key()
+  >>> print(key.decode())
+  ```
 
-  > **NOTE**: Changing the encryption key can be a broken change after service is in use, so make sure you've migrated all the OAuth tokens
-      in the database before replace the key completely, otherwise you'll lose the valid OAuth tokens.
+> **NOTE**: Changing the encryption key can be a broken change after service is in use, so make sure you've migrated all the OAuth tokens in the database before replace the key completely, otherwise you'll lose the valid OAuth tokens.
 
-  ```json
-  "environment_variables": {
-    ...
-    "application_id": "REPLACE_WITH_SANDBOX_APPLICATION_ID",
-    "application_secret": "REPLACE_WITH_SANDBOX_APPLICATION_SECRET",
-    "token_encrypt_secret": "REPLACE_WITH_TOKEN_ENCRYPT_SECRET",
-    ...
-  }
-  ```
+```json
+"environment_variables": {
+  ...
+  "application_id": "REPLACE_WITH_SANDBOX_APPLICATION_ID",
+  "application_secret": "REPLACE_WITH_SANDBOX_APPLICATION_SECRET",
+  "token_encrypt_secret": "REPLACE_WITH_TOKEN_ENCRYPT_SECRET",
+  ...
+}
+```
 
-> **NOTE**: **DO NOT** check in your *application_secret* to git repo.
+> **NOTE**: **DO NOT** check in your _application_secret_ to git repo.
 
 ### Setup and deploy
 
-  1. install [**Chalice**](https://github.com/aws/chalice) and all the project dependencies, **Chalice** is a
-  microframework for writing serverless apps in Python. We use this microframework to manage all the elasticsearch management tasks.
-      ```bash
-      pip install -r requirements.txt
-      ```
+1. install [**Chalice**](https://github.com/aws/chalice) and all the project dependencies, **Chalice** is a
+   microframework for writing serverless apps in Python. We use this microframework to manage all the elasticsearch management tasks.
+   `bash pip install -r requirements.txt `
```

Comment:
`bash` is a syntax highlighter hint (so the markdown parser knows what language this is in) but it should not appear in the actual text that the user copies. You may have to revert this to the previous triple tick format to use it.

Reply:
it's so odd, a lot of these changes got in unintentionally when I opened to read the file :-/ thanks for catching it!